src/py/aspen/database/models/host_filtering.py

```diff
@@ -23,14 +26,26 @@ class HostFilteredSequencingReadsCollection(Entity):
 
     @property
     def sequencing_read(self) -> SequencingReadsCollection:
-        """The raw gisaid dump this processed gisaid dump was generated from."""
+        """The sequencing read collection this host-filtered sequencing read collection
+        was generated from."""
         # this import has to be here for circular dependencies reasons. :(
         from aspen.database.models.sequences import SequencingReadsCollection
 
         parents = self.get_parents(SequencingReadsCollection)
         assert len(parents) == 1
         return parents[0]
 
+    @property
+    def pathogen_genomes(self) -> Sequence[CalledPathogenGenome]:
+        # this import has to be here for circular dependencies reasons. :(
+        from aspen.database.models.sequences import CalledPathogenGenome
```

Comment:
i feel like we should reorganize this so that we don't have to do this (if that's possible)